src/main/java/com/redhat/devtools/lsp4ij/settings/ui/LanguageServerPanel.java

```diff
@@ -79,7 +79,7 @@ public LanguageServerPanel(FormBuilder builder, JComponent description, EditionM
 
     private void createUI(FormBuilder builder, JComponent description, EditionMode mode) {
         JBTabbedPane tabbedPane = new JBTabbedPane();
-        builder.addComponent(tabbedPane);
+        builder.addComponentFillVertically(tabbedPane, 0);

```

Comment:
It's important the the entire tabbed pane be allowed to stretch vertically to accommodate the contents of its tabs.